styling/game.css

```diff
@@ -338,16 +347,21 @@
 }
 
 .boom {
+    pointer-events: none;
     background-image: url('../media/boom.gif') !important;
     background-size: cover;
-    background-position: center;
-    pointer-events: none;
+    background-position: center; 
+}
+
+#computer.gameBoard .boom {
+    background-color: rgb(255, 255, 195);
 }
 
 .empty {
     pointer-events: none;
     background-image: url('../media/empty.gif') !important;
     background-size: cover;
+    background-color: rgb(225, 255, 255);
```

Comment:
Deze kleur is wel heel erg wit. Misschien 50% alpha toevoegen?
Voorbeeld:
`rgba(255,255,255,128)`

Geldt ook voor een hit denk ik.

Reply:
Ik heb de kleuren aangepast denk dat ze nu nog net genoeg op vallen en de gifjes komen ook beter uit.

Reply:
Super